src/main/java/net/sf/jabref/gui/importer/fetcher/OAI2Fetcher.java

```diff
@@ -284,9 +264,11 @@ public boolean processQuery(String query, ImportInspector dialog, OutputPrinter
             }
 
             return true;
-        } catch (Exception e) {
-            status.setStatus(Localization.lang("Error while fetching from %0", "OAI2"));
-            LOGGER.error("Error while fetching from OAI2", e);
+        } catch (IOException | SAXException | InterruptedException e) {
+            LOGGER.error("Error while fetching from " + getTitle(), e);
```

Comment:
In my opinion yes, a `SAXException` won't be resolved by retrying or checking the internet connection.